pallets/messages/src/rpc/src/lib.rs

```diff
@@ -61,13 +61,12 @@ pub enum MessageRpcError {
 	InvalidSchemaId,
 }
 
-impl From<MessageRpcError> for RpcError {
+impl From<MessageRpcError> for JsonRpseeError {
 	fn from(e: MessageRpcError) -> Self {
-		RpcError::Custom(format!("{:?}", e))
+		JsonRpseeError::Custom(format!("{:?}", e))
 	}
 }
 
-#[async_trait]
```

Comment:
I'm surprised we didn't need to take this off of the the other pallets or to have it here. Are we sure this should go away?

Reply:
This is a mystery because it's still in my code locally, GitHub is not reporting any changes yet it's not present on the origin.